cosmo/monitors/nuv_dark_monitor.py

```diff
@@ -0,0 +1,230 @@
+        ax = fig.add_subplot(2, 1, 1)
+        ax.hist(dark_counts, bins=n_bins, align='mid', histtype='stepfilled')
+        counts, bins = np.histogram(dark_counts, bins=100)
+        cuml_dist = np.cumsum(counts)
+        count_99 = abs(cuml_dist / float(cuml_dist.max()) - .99).argmin()
+        count_95 = abs(cuml_dist / float(cuml_dist.max()) - .95).argmin()
+
+        mean = dark_counts.mean()
+        med = np.median(dark_counts)
+        std = dark_counts.std()
+        mean_obj = ax.axvline(x=mean, lw=2, ls='--', color='r', label='Mean ')
+        med_obj = ax.axvline(x=med, lw=2, ls='-', color='r', label='Median')
+        two_sig = ax.axvline(x=med + (2 * std), lw=2, ls='-', color='gold')
+        three_sig = ax.axvline(x=med + (3 * std), lw=2, ls='-',
+                               color='DarkOrange')
+        dist_95 = ax.axvline(x=bins[count_95], lw=2, ls='-',
+                             color='LightGreen')
+        dist_99 = ax.axvline(x=bins[count_99], lw=2, ls='-', color='DarkGreen')
+
+        ax.grid(True, which='both')
+        ax.set_title('Histogram of Dark Rates', fontsize=15, fontweight='bold')
+        ax.set_ylabel('Frequency', fontsize=15, fontweight='bold')
+        ax.set_xlabel('Counts/pix/sec', fontsize=15, fontweight='bold')
+        ax.set_xlim(dark_counts.min(), dark_counts.max())
+        ax.xaxis.set_major_formatter(FormatStrFormatter('%3.2e'))
+
+        ax = fig.add_subplot(2, 1, 2)
+        # log_bins = np.logspace(np.log10(dark.min()), np.log10(dark.max()),
+        # 100)
+        ax.hist(dark_counts, bins=n_bins, align='mid', log=True,
+                histtype='stepfilled')
+
+        ax.axvline(x=mean, lw=2, ls='--', color='r', label='Mean')
+        ax.axvline(x=med, lw=2, ls='-', color='r', label='Median')
+        ax.axvline(x=med + (2 * std), lw=2, ls='-', color='gold')
+        ax.axvline(x=med + (3 * std), lw=2, ls='-', color='DarkOrange')
+        ax.axvline(x=bins[count_95], lw=2, ls='-', color='LightGreen')
+        ax.axvline(x=bins[count_99], lw=2, ls='-', color='DarkGreen')
+
+        # ax.set_xscale('log')
+        ax.grid(True, which='both')
+        ax.set_ylabel('Log Frequency', fontsize=15, fontweight='bold')
+        ax.set_xlabel('Counts/pix/sec', fontsize=15, fontweight='bold')
+        ax.set_xlim(dark_counts.min(), dark_counts.max())
+        ax.xaxis.set_major_formatter(FormatStrFormatter('%3.2e'))
+
+        fig.legend([med_obj, mean_obj, two_sig, three_sig, dist_95, dist_99],
+                   ['Median: {0:.2e}'.format(med),
+                    'Mean: {0:.2e}'.format(mean),
+                    r'2$\sigma$: {0:.2e}'.format(med + (2 * std)),
+                    r'3$\sigma$: {0:.2e}'.format(med + (3 * std)),
+                    r'95$\%$: {0:.2e}'.format(bins[count_95]),
+                    r'99$\%$: {0:.2e}'.format(bins[count_99])], shadow=True,
+                   numpoints=1, bbox_to_anchor=[0.8, 0.8])
```

Comment:
Just to reiterate, plotting should not be in `track`, but also, `plotly` should be used for plotting if possible by creating traces and whatnot and then adding them to the `figure` attribute: 

```python
    def plot(self):
        ...  # Make plotly traces here; let's say you have a list called "traces"
        self.figure.add_traces(traces)
```

Reply:
yeah that is the end goal as well, but marc wanted the plots replicated for now